lib/goo/sparql/client.rb

```diff
@@ -81,26 +81,27 @@ def delete_data_graph(graph)
       end
 
       def append_triples_no_bnodes(graph,file_path,mime_type_in)
-        bnodes_filter = nil
         dir = nil
         response = nil
         if file_path.end_with?('ttl')
           bnodes_filter = file_path
+          file = File.foreach(bnodes_filter)
```

Comment:
File.read must be used instead of File.foreach here; file content is not getting loaded.

Reply:
I updated it